pom.xml

```diff
@@ -16,7 +16,18 @@
 				<maven.compiler.source>1.8</maven.compiler.source>
 				<maven.compiler.target>1.8</maven.compiler.target>
 		</properties>
-
+<build>
+		<resources>
+			<resource>
+				<directory>src/main/resources/</directory>
+				<includes>
+					<include>*.yml</include>
+					<include>LICENSE.txt</include>
```

Comment:
License is in the base directory (where it should be) and it needs it's own entry as resource to be included. This would only work if the license was under src/main/resources

Reply:
Yeah this is copypasta from some other plugin.